io-engine/src/bdev/nexus/nexus_bdev_snapshot.rs

```diff
@@ -191,19 +180,65 @@ impl ReplicaSnapshotExecutor {
                     Some(ctx.snapshot_uuid.clone()),
                     snapshot.create_time(),
                 );
+
                 let replica_uuid = ctx.replica_uuid.clone();
-                debug!(
-                    replica_uuid,
-                    ?snapshot_params,
-                    "Starting nexus replica snapshot operation",
-                );
+                let nexus_name = self.nexus_name.clone();
+
+                // Schedule replica snapshot operation on master core.
```

Comment:
Aren't we already on the master core here?
We seem to already be doing things like pausing the subsystem which has an assert for the master core.